cmd/antrea-agent/options.go

```diff
@@ -54,7 +54,9 @@ type Options struct {
 
 func newOptions() *Options {
 	return &Options{
-		config: new(AgentConfig),
+		config: &AgentConfig{
```

Comment:
I remember there were some misleading code about this but forgot to correct them.
Although we initialize `config` here, it was discarded in `complete`, so I guess setting the default value here doesn't take effect. We should change L139-L140 as well to use the initialized `config`.  So do antrea-controller.

Reply:
Just realized that too because of a test failure, making the change

Reply:
Done, I tried to improve the code a bit there as well